plugins/modules/cloud/opennebula/one_template.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/python
+#
+# Copyright: (c) 2021, Georg Gadinger <nilsding@nilsding.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: one_template
+
+short_description: Manages OpenNebula templates
+
+requirements:
+  - pyone
+
+description:
+  - "Manages OpenNebula templates"
+
+options:
+  id:
+    description:
+      - A C(id) of the template you would like to manage.  If not set then a
+      - new template will be created with the given C(name).
+    type: int
+  name:
+    description:
+      - A C(name) of the template you would like to manage.  If a template with
+      - the given name does not exist it will be created, otherwise it will be
+      - managed by this module.
+    type: str
+  template:
+    description:
+      - A string containing the template contents.
+    type: str
+  state:
+    description:
+      - C(present) - state that is used to manage the template
+      - C(absent) - delete the template
+    choices: ["present", "absent"]
+    default: present
+    type: str
+
+extends_documentation_fragment:
+- community.general.opennebula
+
+author:
+    - "Georg Gadinger (@nilsding)"
```

Comment:
Hi @nilsding , please use a consistent indentation throughout the documentation, it improves the readability.